src/base/settingsstorage.cpp

```diff
@@ -1,7 +1,7 @@
 /*
  * Bittorrent Client using Qt and libtorrent.
  * Copyright (C) 2016  Vladimir Golovnev <glassez@yandex.ru>
- * Copyright (C) 2014  sledgehammer999 <hammered999@gmail.com>
+ * Copyright (C) 2014  sledgehammer999 <sledgehammer999@qbittorrent.org>
```

Comment:
Is this email really working?

Reply:
Probably, i searched for some issues where sledgehammer answered and this is the email he gave to the users e.g. https://github.com/qbittorrent/qBittorrent/issues/8080#issuecomment-353222306.